CHANGELOG.md

```diff
@@ -21,6 +21,21 @@ recommendations of [keepachangelog.com](http://keepachangelog.com/).
 
 - None
 
+### Fixed
+
+- None
+
+## 8.1.1 (2017-12-02)
```

Comment:
Please put your changelog entry under "unreleased". Do not specify a version number. I'll set the version number when I do the release.